moveit_simple/CMakeLists.txt

```diff
@@ -87,6 +89,10 @@ if(CATKIN_ENABLE_TESTING)
   catkin_add_gtest(moveit_simple_utest ${MOVEIT_SIMPLE_TEST_SRC_FILES})
   target_link_libraries(moveit_simple_utest ${PROJECT_NAME} ${catkin_LIBRARIES} ${Boost_LIBRARIES})
 
+  set(TOOL_WINDUP_SRC_FILES test/tool_windup_utest.cpp test/tool_windup.cpp)
```

Comment:
Add comment describing what this does

Reply:
Inside the test source or directly inside the CMakeLists?